app/views/conditions/edit.erb

```diff
@@ -0,0 +1,78 @@
+<% @title = "Edit Weather Info" %>
```

Comment:
anytime you find yourself setting an instance variable in your views, it should be an indication that you're doing too much in your views. your view's responsibility is to simply display data